modules/node-shared/src/test/resources/seedlists/seedlist.5fields.invalid.sample

```diff
@@ -0,0 +1,9 @@
+3458a688925a4bd89f2ac2c695362e44d2e0c2903bdbb41b341a4d39283b22d8c85b487bd33cc5d36dbe5e31b5b00a10a6eab802718ead4ed7192ade5a5d1941,233.43.36.232,test1,1,0--100
+46daea11ca239cb8c0c8cdeb27db9dbe9c03744908a8a389a60d14df2ddde409260a93334d74957331eec1af323f458b12b3a6c3b8e05885608aae7e3a77eac7,33.82.174.165,test2,0,0-100;-50-50
+46daea11ca239cb8c0c8cdeb27db9dbe9c03744908a8a389a60d14df2ddde409260a93334d74957331eec1af323f458b12b3a6c3b8e05885608aae7e3a77eac7,33.82.174.165,test2,0,0-49;-50-50
+e2f4496e5872682d7a55aa06e507a58e96b5d48a5286bfdff7ed780fa464d9e789b2760ecd840f4cb3ee6e1c1d81b2ee844c88dbebf149b1084b7313eb680714,107.114.18.119,test3,-1,0-49;50-50;60-51
+ac5030e30ccf3c50b52b092c5b2b3a85ca13f72f85571b74f1ec062db7993e9d1c87edbb79b1aeed6038752fd6655dfd4995a0807067db2f20c1a63892d8f14c,189.122.117.43,test4,0.5,0-49;50-50;60-61;80-79
+ac5030e30ccf3c50b52b092c5b2b3a85ca13f72f85571b74f1ec062db7993e9d1c87edbb79b1aeed6038752fd6655dfd4995a0807067db2f20c1a63892d8f14c,189.122.117.43,test4,0.5,0-49;5 0-50;60-61
+ac5030e30ccf3c50b52b092c5b2b3a85ca13f72f85571b74f1ec062db7993e9d1c87edbb79b1aeed6038752fd6655dfd4995a0807067db2f20c1a63892d8f14c,189.122.117.43,test4,0.5,0-49;50- 50;60-61
```

Comment:
why this is invalid? 🤔

Reply:
1. `60-51` is invalid because `start > end`.
2. `80-79` is invalid because `start > end`.
3. `5 0-50` is invalid because `5 0` is not a number.
4. This one is not invalid. The file has been scrubbed of all valid entries.